response-body-view.html

```diff
@@ -98,7 +99,27 @@
       right: 12px;
       background: #fff;
       color: rgba(0,0,0,0.74);
-      @apply(--response-body-preview-close);
+    }
+    .dialog-title {
```

Comment:
Please, use `--paper-dialog-title` mixin to style this element. It's more scalable because it will the styling apply to all titles

Reply:
thanks